pallas-network/src/multiplexer.rs

```diff
@@ -84,25 +89,45 @@ impl Bearer {
         Ok((Self::Tcp(stream), addr))
     }
 
+    #[cfg(windows)]
+    pub async fn connect_pipe(pipe_name: impl AsRef<std::ffi::OsStr>) -> Result<Self, tokio::io::Error>{
```

Comment:
could we rename the fn as `connect_named_pipe`?

Reply:
code is updated to "connect_named_pipe" but im still seeing it as "connect_pipe" in the comment here for some reason